ulog/builder.go

```diff
@@ -87,6 +96,7 @@ func (lb *LogBuilder) WithSentryHook(hook *sentry.Hook) *LogBuilder {
 }
 
 // Build the ulog logger for use
+// TODO: build should return `(Log, error)` in case we can't properly instantiate
```

Comment:
Didn't want to bloat the PR and touch that code, but will at some point. We do some unmarhsaling, we read configs and do some other things that produce errors in logger build. When encountering error, we log an error message and fall back to default value. I think we should probably just return those errors to make sure service owners don't misconfigure logging by accident